templates/nvidia.yaml

```diff
@@ -26,19 +26,16 @@ spec:
   chartReference:
     chart: nvidia
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.1.2
+    version: 0.2.0
     values: |
       ---
       nvidia-driver:
         image:
           tag: "418.87.01-centos7"
         resources:
```

Comment:
@hectorj2f You've done a ton of work on resource parameters. Do you have any issue with this change?

Reply:
Are you asking for why we remove the limits?

Because:
1. the driver can burst up to 8GB or more in the first 5 min. It is compiling kernel modules during runtime. Every time memory is throttled by the limits, the driver might be OOM killed.
2. GPU is expensive resources and we only run this driver on gpu machines. Without the driver up and running, there is no GPU jobs can be launched. We don't want to affect the GPU base driver due to memroy limits.

Basically, for mission critical applications, it should be ok to set its QoS as burstable without limits.